store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonSchemaReader.java

```diff
@@ -144,4 +246,28 @@ public static Schema readSchemaInIndexFile(String indexFilePath) throws IOExcept
     }
   }
 
+  /**
+   * This method return the version details in formatted string by reading from carbondata file
+   *
+   * @param dataFilePath
+   * @return
+   * @throws IOException
+   */
+  public static String getVersionDetails(String dataFilePath) throws IOException {
```

Comment:
This complete method is displayed as removed and added again. Is it possible to avoid?

Reply:
can't